components/identity-core/org.wso2.carbon.identity.core/src/test/resources/testng.xml

```diff
@@ -28,6 +28,7 @@
             <class name="org.wso2.carbon.identity.core.internal.DefaultServiceURLBuilderTest"/>
             <class name="org.wso2.carbon.identity.core.cache.BaseCacheTest"/>
             <class name="org.wso2.carbon.identity.core.ThreadLocalAwareThreadPoolExecutorTest"/>
+            <class name="org.wso2.carbon.identity.core.IdentityKeyStoreResolverTest"/>
```

Comment:
What about the util test class?

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/5794/commits/6d6ea654f817ebeaeb86f5ed0a17a1d001e49806